### PR TITLE
Feature/job pipelines

### PR DESCRIPTION
**Pull request by Lance-Simeon**
Update installer script to support summary and retry pipelines from the jobs repo. Once this PR is approved, I will also be merging the Org Install to master.

### CODE REVIEW

File: SimeonInstaller.ps1 [hunk 1]
```diff
@@ -1217,6 +1217,82 @@ CRLFOption=CRLFAlways
...
+        Push-Location (Get-GitRepository $GitSourceUrl)
+
+        Write-Information "Deleting existing .git directory"
+        Remove-Item '.\.git' -Recurse -Force
+
+        Write-Verbose "Initializing new git repository with existing contents"
+        Invoke-CommandLine "git init 2>&1" | Write-Verbose
+        Initialize-GitConfiguration
+
+        $gitConfig = "-c http.extraheader=`"AUTHORIZATION: bearer $token`""
```
Comment (admin-simeon) [inline SimeonInstaller.ps1:1281]: perhaps we should set this in git config inside Get-GitRepository
```
git config http.extraheader=`"AUTHORIZATION: bearer $token`"
```

File: SimeonInstaller.ps1 [hunk 2]
```diff
@@ -1217,6 +1217,82 @@ CRLFOption=CRLFAlways
...
+                } | ConvertTo-Json)
+        }
+        else {
+            Write-Verbose "Repository already exists - will not create"
+            return $repo
+        }
+
+        Write-Information "Installing repo: $RepoName for $organization"
+
+        Push-Location (Get-GitRepository $GitSourceUrl)
```
Comment (admin-simeon) [inline SimeonInstaller.ps1:1272]: always use try/finally with Push/Pop location

File: SimeonInstaller.ps1 [hunk 3]
```diff
@@ -1217,6 +1217,82 @@ CRLFOption=CRLFAlways
...
+                        name = $Project
+                    }
+                } | ConvertTo-Json)
+        }
+        else {
+            Write-Verbose "Repository already exists - will not create"
+            return $repo
+        }
+
+        Write-Information "Installing repo: $RepoName for $organization"
```
Comment (admin-simeon) [inline SimeonInstaller.ps1:1270]:
```
        Write-Information "Installing repository '$RepoName' for '$Organization'"
```

terms are quoted and $Organization capitalization was wrong

File: SimeonInstaller.ps1 [hunk 4]
```diff
@@ -1217,6 +1217,82 @@ CRLFOption=CRLFAlways
 "@ | Out-Null
     }
 
+    <#
+    .SYNOPSIS
+    Clones git repo into Azure an DevOps project
+    #>
+    function Install-DevOpsRepoFromGitHub {
```
Comment (admin-simeon) [inline SimeonInstaller.ps1:1224]: `Install-SimeonTenantRepository` should use this, no?

Comment (admin-simeon) [reply SimeonInstaller.ps1:1224]: `Install-DevOpsRepositoryFromGitHub`

**Review by admin-simeon — CHANGES_REQUESTED** [review]
👍

File: SimeonInstaller.ps1 [hunk 5]
```diff
@@ -562,9 +562,9 @@ CRLFOption=CRLFAlways
     function Get-AuthenticationHeader {
```
[the remark this answers is not in the document]
Comment (Lance-Simeon) [reply SimeonInstaller.ps1:562]: Don't remember talking about it, this function is only used for the org install for the moment. If we want to support PAT tokens beyond the org installer, we can expand everything to use this function. Was trying to keep changes to a minimum for this effort

**Comment by JeffN825** [comment]
But doesn’t that encoding and way of basic auth work for bearer tokens too? I thought you had inadvertently tried it and found it to work?

________________________________

File: SimeonInstaller.ps1 (hunk 5 above)
[the remark this answers is not in the document]
Comment (admin-simeon) [reply SimeonInstaller.ps1:562]: But doesn’t that encoding and way of basic auth work for bearer tokens too? I thought you had inadvertently tried it and found it to work?

**Pull request merged**